client/packages/invoices/src/Returns/InboundDetailView/SidePanel/RelatedDocumentsSection.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { memo } from 'react';
+import {
+  Grid,
+  DetailPanelSection,
+  PanelField,
+  PanelLabel,
+  PanelRow,
+  useTranslation,
+  Tooltip,
+  Link,
+  useFormatDateTime,
+  RouteBuilder,
+} from '@openmsupply-client/common';
+import { AppRoute } from '@openmsupply-client/config';
+import { useReturns } from '../../api';
+
+export const RelatedDocumentsSectionComponent = () => {
+  const t = useTranslation('distribution');
+  const { localisedDate: d } = useFormatDateTime();
+  const { data } = useReturns.document.inboundReturn();
+  const { originalShipment } = data ?? {};
+
+  let tooltip = '';
```

Comment:
hmm, I think tooltips don't really work for mobile users (at least it's not apparent that they exist).
I wonder if we just remove this or add the 'info' icon (to suggest it's pressable). Or even include this info as the label ?

Reply:
![Screenshot 2024-03-27 at 4 24 32 PM](https://github.com/msupply-foundation/open-msupply/assets/26194949/b698d8ef-8162-44c8-99a7-c4769c63d371)

Reply:
Ah yeah let's just include the info in the label - was a copy paste but I like the info being more easily available, and it's not like we're short on space in that side panel at the moment! 😁